docs/how-to-install.md

```diff
@@ -123,7 +123,7 @@ Gravitino publishes the Docker image to [Docker Hub](https://hub.docker.com/r/da
 You can run the Gravitino Docker image by:
 
 ```shell
-docker run -d -i -p 8090:8090 datastrato/gravitino:0.3.0-SNAPSHOT
+docker run -d -i -p 8090:8090 datastrato/gravitino:<version>
```

Comment:
How do you know what versions exist? Might be good to add a note on how to find them?

Reply:
I describe the gravitino docker hub address above, and we will have a docker image release for each version, user could find them on the docker hub.